client/my-sites/plan-features/header.jsx

```diff
@@ -155,25 +144,6 @@ export class PlanFeaturesHeader extends Component {
 		);
 	}
 
-	renderPlansHeaderP2Free() {
-		const { planType, isInSignup, translate } = this.props;
-
-		const headerClasses = classNames( 'plan-features__header', getPlanClass( planType ), {
-			'is-p2-free': true,
```

Comment:
Same as above :)

https://github.com/Automattic/wp-calypso/blob/0c16202f7a022efff85d1097bcbd4818b7df869b/client/my-sites/plan-features/style.scss#L314

Reply:
Yes! Thank you 😃

Reply:
https://github.com/Automattic/wp-calypso/pull/83320/commits/4a3248ad4da05de96e6ccc8aa1931ff13371b7ca

Reply:
https://github.com/Automattic/wp-calypso/commit/4a3248ad4da05de96e6ccc8aa1931ff13371b7ca